qiskit/circuit/library/arithmetic/adder/__init__.py

```diff
@@ -0,0 +1,16 @@
+# This code is part of Qiskit.
```

Comment:
Could you delete all the files in the old `adder` folder (now that the code is in the `adders` directory)? 🙂